test/TerminalTest.py

```diff
@@ -0,0 +1,296 @@
+import pkg_resources
+import unittest
+
+import Adyen
+
+try:
+    from BaseTest import BaseTest
+except ImportError:
+    from .BaseTest import BaseTest
+
+VERSION = pkg_resources.get_distribution("Adyen").version
```

Comment:
Not sure if I should use the version from the settings or the one from the package installed :thinking: